src/shared.ts

```diff
@@ -102,6 +117,14 @@ const createWrapper = (stripeConnect: any) => {
           }
         }
       });
+      const oldCreate = stripeConnectInstance.create.bind(
```

Comment:
What is the purpose of this?

Reply:
`create` on ConnectInstance only takes in HTML names, we basically want to wrap that method so we can pass in the simplified tag name instead. However if we don't bind the method we run into a recursive error since we're basically replacing the create method and calling it at the same time if we only do something like 

`stripeConnectInstance.create = (tagName: ConnectElementTagName) => {
        const htmlName = componentNameMapping[tagName];
        stripeConnectInstance.create(htmlName);
      };`